src/SixLabors.Fonts/FontCollectionExtensions.cs

```diff
@@ -17,12 +19,31 @@ public static FontCollection AddSystemFonts(this FontCollection collection)
         {
             // This cast is safe because our underlying SystemFontCollection implements
             // both interfaces separately.
-            foreach (FontMetrics? metric in (IReadOnlyFontMetricsCollection)SystemFonts.Collection)
+            foreach (FontMetrics metric in (IReadOnlyFontMetricsCollection)SystemFonts.Collection)
             {
                 ((IFontMetricsCollection)collection).AddMetrics(metric);
             }
 
             return collection;
         }
+
+        /// <summary>
+        /// Adds the fonts from the <see cref="SystemFonts"/> collection to this <see cref="FontCollection"/>.
+        /// </summary>
+        /// <param name="collection">The font collection.</param>
+        /// <param name="match">The System.Predicate delegate that defines the conditions of <see cref="FontMetrics"/> to add into the font collection.</param>
```

Comment:
```suggestion
        /// <param name="match">The <see cref="System.Predicate"/> delegate that defines the conditions of <see cref="FontMetrics"/> to add into the font collection.</param>
```

Reply:
CI error fixed. Should be `<see cref="Predicate{T}"/>`.